components/org.wso2.carbon.identity.api.user.association/org.wso2.carbon.identity.api.user.association.v1/src/main/resources/association.yaml

```diff
@@ -624,6 +667,89 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Property'
+
+  BulkFederatedAssociationRequest:
+    type: object
+    properties:
+      failOnErrors:
+        type: integer
+        example: 1
+        description: |
+          This represents whether the API should stop processing the operations if any of them fail.
+          If set to 0 or not set, the API will continue processing the operations even if some of them fail.
+      operations:
+        type: array
+        items:
+          $ref: '#/definitions/BulkFederatedAssociationOperation'
+
+  BulkFederatedAssociationOperation:
+    type: object
+    required:
+      - method
+      - bulkId
+      - path
+    properties:
+      method:
+        type: string
```

Comment:
lets switch to enum

Reply:
e6f9ea1307154505fa3b88615be9f717e1dcacfd